onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -1587,6 +1595,32 @@ bool ResizeOpSupportChecker::IsOpSupportedImpl(const InitializedTensorSet& initi
       }
     }
   }
+
+  if (IsQuantizedOp(node_unit)) {
+    // For QDQResize, we only support uint8 output now
+    int32_t output_type;
+    if (!GetType(node_unit.Outputs()[0].node_arg, output_type))
+      return false;
+
+    if (output_type != ONNX_NAMESPACE::TensorProto_DataType_UINT8) {
+      LOGS_DEFAULT(VERBOSE) << "[Resize] output type: [" << output_type
+                            << "] is not supported for now";
+      return false;
+    }
+
+    // Check input scales and ZPs
+    if (!HasValidQuantizationScales(initializers, node_unit, {0}, params, true /* is_input */))
+      return false;
+    if (!HasValidQuantizationZeroPoints(initializers, node_unit, {0}, true /* is_input */))
+      return false;
+
+    // Check output scale and ZP
+    if (!HasValidQuantizationScales(initializers, node_unit, {0}, params, false /* is_input */))
+      return false;
+    if (!HasValidQuantizationZeroPoints(initializers, node_unit, {0}, false /* is_input */))
+      return false;
+  }
```

Comment:
As this is also validating inputs/output maybe it could be called something like IsValidQuantizedOp.

Reply:
Or maybe all the validation should be in a separate func given we seem to do some here and some in ResizeOpBuilder::AddToModelBuilderImpl

Reply:
Yes, there will be a separated change to merge all these HasValidQuantizationScales and HasValidQuantizationZeroPoints into one single helper function, plus the input and output type check,
Will be added in a later PR.

The validation of quantized IO params are different in SupportChecker and OPbuilder, the one in SupportChecker only checks if the quantized scale and zp are initializers and some other checks such as per-channel setting, the one in OPBuilder will also verify if these quantization params are matching the existing numbers in the NNAPI model, this part cannot be done in SupportChecker.
Will figure out better names for these in a later PR